### PR TITLE
Upgrade to Swift 5

### PR DESCRIPTION
**Pull request by ricky641b**
- updated all the libraries
- moved all dependencies to cocoapods
- upgraded to Swift 5
- fixed DB errors and crashes after the upgrade

### CODE REVIEW

**Comment by MilapNaik** [comment]
I'm getting errors like an error from import GoogleMobileAds and import FirebaseAnalytics in FinishTest

**Comment by ricky641b** [comment]
Please install cocoapods from the terminal - Follow this link [Here](https://cocoapods.org/)
After installing navigate to the project directory from the Terminal and run `pod install`

**Comment by MilapNaik** [comment]
These steps finally solved my issue. Must have been the temp files.

https://developer.apple.com/forums/thread/128535

File: TraderMathTest/Vendor/SQLite/CloudDB.swift [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+		let data = row.values()
+		let idName = row.remoteKey()
+		let record: CKRecord
+		if let ckid = recordID {
+			record = CKRecord(recordType: row.table, recordID: ckid)
+		} else {
+			if type == .privateDB {
+//				let zone = CKRecordZone.ID(zoneName: row.table, ownerName: CKCurrentUserDefaultName)
+//				let ckid = CKRecord.ID(recordName: "", zoneID: zone)
+//				record = CKRecord(recordType: row.table, recordID: ckid)
```
Comment (MilapNaik) [inline TraderMathTest/Vendor/SQLite/CloudDB.swift:176]: Is this commented code necessary

File: TraderMathTest/Helper/Extension/String-Extras.swift [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+//    }
+//
+//    func urlDecoded()->String {
+//        let res:NSString = CFURLCreateStringByReplacingPercentEscapesUsingEncoding(kCFAllocatorDefault, self as NSString, "" as CFString, CFStringConvertNSStringEncodingToEncoding(String.Encoding.utf8.rawValue))
+//        return res as String
+//    }
+//
+//    func range()->Range<Index> {
+//        return Range<Index>(startIndex..<endIndex)
+//    }
```
Comment (MilapNaik) [inline TraderMathTest/Helper/Extension/String-Extras.swift:52]: Is this code necessary

Comment (ricky641b) [reply TraderMathTest/Helper/Extension/String-Extras.swift:52]: We are not using this code anywhere. So I commented this code.

File: TraderMathTest/Vendor/SQLite/CloudDB.swift (hunk 1 above)
Comment (ricky641b) [reply TraderMathTest/Vendor/SQLite/CloudDB.swift:176]: No its not necessary

**Pull request merged**